src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -303,7 +303,7 @@ protected virtual IObjectGraphType ToObjectGraphType(GraphQLObjectTypeDefinition
             return type;
         }
 
-        private void InitializeField(FieldConfig config, Type? parentType)
+        private void InitializeField(FieldConfig config, [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicMethods | DynamicallyAccessedMemberTypes.PublicProperties)] Type? parentType)
```

Comment:
This is for schema-first.

Reply:
Do you mean that for entire schema-first parts of project we should not bother with all these annotations and just mark all top-level public classes with `[RequiresUnreferencedCode]`?